### PR TITLE
upgrade to androidx , remove local dependencies to gradle.

### PR DESCRIPTION
**Pull request by mohamedagamy**
chore:  I have updated the sample project to depend on gradle instead of local project modules.
I have migrated the sample to androidx.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -33,31 +33,30 @@
         </value>
       </option>
     </JavaCodeStyleSettings>
-    <Objective-C-extensions>
```
Comment (rantianhua) [inline .idea/codeStyles/Project.xml]: Please don't change this file.

File: .idea/modules.xml [hunk 2]
```diff
@@ -3,13 +3,22 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/benchmark/benchmark.iml" filepath="$PROJECT_DIR$/benchmark/benchmark.iml" />
+      <module fileurl="file://$PROJECT_DIR$/benchmark/benchmark.iml" filepath="$PROJECT_DIR$/benchmark/benchmark.iml" />
```
Comment (rantianhua) [inline .idea/modules.xml]: Please don't change this file.

**Comment by mohamedagamy** [comment]
ok i think if you merge there updates without the both files it will be ok
there are android studio config files.

**Comment by mohamedagamy** [comment]
I have no idea how to avoid this can you help me

**Comment by rantianhua** [comment]
You can ignore this idea file during you commit. Please provide an new pull request for androidx because of the sample have a lot of changes after upgrade to 1.0.6.

**Pull request closed without merging**